reporter/slack.go

```diff
@@ -35,9 +35,9 @@ type message struct {
 
 func (w SlackWriter) Write(rs ...models.ScanResult) (err error) {
 
-	channel := w.Cnf.Channel
 	for _, r := range rs {
 		w.lang, w.osFamily = r.Lang, r.Family
+		channel := w.Cnf.Channel
 		if channel == "${servername}" {
```

Comment:
If the scope of the channel variable is to be considered, then this is the way to do it.
```suggestion
		if w.Cnf.Channel == "${servername}" {
```

Reply:
> Therefore, I think we should evaluate whether w.Cnf.Channel is ${servername} or not.
Yes. At first, I tried the method of comparing w.Cnf.Channel and ${servername}.
Of course, it was working.
However the channel parameter is used only on for loop, then I moved   channel parameter declaration within for loop.
I believe this is a coding policy, so choose the best for you.

Reply:
> Therefore, it seems more appropriate to compare ${servername} in w.Cnf.Channel, where config.toml is read in, rather than in channel.

Nothing. I think too that it is better to use w.Cnf.Channel than channel parameter when comparing ${servername}.

Reply:
Okay, I changed it to use w.Cnf.Channel.